game/draw.lua

```diff
@@ -24,6 +24,8 @@ function draw.allTables()
 
     CAM:detach() --Stop tracking camera
 
+    DrawTable(DRAW_TABLE.HUDl)
```

Comment:
was it really necessary to create a whole new layer for this? the regular hud layer should have been fine, no?

Reply:
The problem is that there is no guaranteed drawing order between elements in the same layer. In this case, @rilifon wanted to add a transluscent background to the HUD and, in order to make sure it is drawn at the right time, the only way is to rely on the layer mechanism.

However, searching the code for "L2" shows that this layer is still unused. Should we use it instead? Won't we need it for anything else?

Reply:
ah okay, but we should rename them so they make sense. like, why is HUD1 drawn before HUD? we can do this in a different PR tho

Reply:
its not HUD1, its HUDl for lower. I made a coment about it on the setup.lua file.

@Kazuo256 comments explains my line of thought